README.md

```diff
@@ -36,7 +67,7 @@ To run these network integration tests, use ansible-test network-integration --i
 
 ## Publishing New Version
 
-The current process for publishing new versions of the Cisco ASA Community Collection is manual, and requires a user who has access to the `community` namespace on Ansible Galaxy to publish the build artifact.
+The current process for publishing new versions of the Community Collection is manual, and requires a user who has access to the `community` namespace on Ansible Galaxy to publish the build artifact.
```

Comment:
This section requires to be updated to reflect the automated collection build and publish process to galaxy by tagging the version